src/components/LocationStepper.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+
+import { SELECTED, HOVERED } from '../reducers/selectionReducer';
+import { PRODUCT_SELECT, APPLY } from '../router';
+import * as colors from '../helpers/colors';
+import Stepper from '../Stepper';
+
+const stepMap = {
+  [PRODUCT_SELECT]: 0,
+  [APPLY]: 1
+};
+
+const getSelectedProducts = (selectedProducts) => {
+  return _.reduce(selectedProducts, (result, state) => {
+    if(state === SELECTED || state === HOVERED){
+      return result + 1;
+    }
+    return result;
+  }, 0);
+};
+
+const LocationStepper = ({location, selectedProducts}) => {
+  const activeStep = stepMap[location];
+  const numOfProducts = getSelectedProducts(selectedProducts);
+  const stepOneTitle = (location === PRODUCT_SELECT) ? 'SELECT PRODUCTS' : `SELECT PRODUCTS (${numOfProducts})`;
```

Comment:
I know the `numOfProducts` was in the mockup, but let's get rid of it. Kind of pointless to have that there.